src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -90,7 +92,7 @@ class Engine(Enum):  # pylint: disable=too-few-public-methods
 
 
 def _configure_db_create_params(
```

Comment:
Can you please check that all these db commands have exactly the same params before and after this change?

Reply:
It is actually not right now, because of #5037. I have updated #4919 with this check point